clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/security/OpenstackNamedAccountCredentials.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2016 Veritas Technologies LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.openstack.security;
+
+import com.netflix.spinnaker.clouddriver.security.AccountCredentials;
+
+import java.util.List;
+
+public class OpenstackNamedAccountCredentials implements AccountCredentials<OpenstackCredentials> {
+  private static final String CLOUD_PROVIDER = "openstack";
+  private final String accountName;
+  private final String environment;
+  private final String accountType;
+  private final String master;
+  private final String username;
+  private final String password;
```

Comment:
this class is going to get serialized out the /credentials endpoint, so anything sensitive in here should get `@JsonIgnore`'d